files/jenkins.json

```
@@ -1,33 +1,34 @@
-        { "name": "cloudbees-folder", "version": "6.1012.v79a_86a_1ea_c1f" },
-        { "name": "display-url-api", "version": "2.209.v582ed814ff2f" },
-        { "name": "durable-task", "version": "587.v84b_877235b_45" },
-        { "name": "instance-identity", "version": "201.vd2a_b_5a_468a_a_6" },
-        { "name": "ionicons-api", "version": "82.v0597178874e1" },
+        { "name": "asm-api", "version": "9.9-185.va_6c6b_3348b_c3" },
+        { "name": "bouncycastle-api", "version": "2.30.1.82-277.v70ca_0b_877184" },
+        { "name": "branch-api", "version": "2.1259.v45c101731c76" },
+        { "name": "caffeine-api", "version": "3.2.3-194.v31a_b_f7a_b_5a_81" },
+        { "name": "cloudbees-folder", "version": "6.1073.va_7888eb_dd514" },
+        { "name": "commons-lang3-api", "version": "3.19.0-104.v12125f33a_255" },
+        { "name": "display-url-api", "version": "2.217.va_6b_de84cc74b_" },
+        { "name": "durable-task", "version": "635.v3733cef34b_5e" },
+        { "name": "instance-identity", "version": "203.v15e81a_1b_7a_38" },
+        { "name": "ionicons-api", "version": "94.vcc3065403257" },
         { "name": "jakarta-activation-api", "version": "2.1.3-2" },
-        { "name": "jakarta-mail-api", "version": "2.1.3-2" },
+        { "name": "jakarta-mail-api", "version": "2.1.3-3" },
         { "name": "javax-activation-api", "version": "1.2.0-8" },
         { "name": "javax-mail-api", "version": "1.6.2-11" },
-        { "name": "mailer", "version": "489.vd4b_25144138f" },
-        { "name": "pipeline-groovy-lib", "version": "752.vdddedf804e72" },
-        { "name": "scm-api", "version": "704.v3ce5c542825a_" },
-        { "name": "script-security", "version": "1373.vb_b_4a_a_c26fa_00" },
-        { "name": "structs", "version": "343.vdcf37b_a_c81d5" },
+        { "name": "mailer", "version": "522.va_995fa_cfb_8b_d" },
+        { "name": "pipeline-groovy-lib", "version": "776.vfee5327b_b_a_5b_" },
+        { "name": "scm-api", "version": "712.v8846fdd68c88" },
+        { "name": "script-security", "version": "1385.v7d2d9ec4d909" },
+        { "name": "structs", "version": "353.v261ea_40a_80fb_" },
         { "name": "variant", "version": "70.va_d9f17f859e0" },
-        { "name": "workflow-api", "version": "1371.ve334280b_d611" },
-        { "name": "workflow-basic-steps", "version": "1079.vce64b_a_929c5a_" },
-        { "name": "workflow-cps", "version": "4046.v90b_1b_9edec67" },
-        { "name": "workflow-durable-task-step", "version": "1405.v1fcd4a_d00096" },
-        { "name": "workflow-job", "version": "1520.v56d65e3b_4566" },
-        { "name": "workflow-multibranch", "version": "806.vb_b_688f609ee9" },
-        { "name": "workflow-scm-step", "version": "437.v05a_f66b_e5ef8" },
-        { "name": "workflow-step-api", "version": "700.v6e45cb_a_5a_a_21" },
-        { "name": "workflow-support", "version": "968.v8f17397e87b_8" }
+        { "name": "workflow-api", "version": "1398.v67030756d3fb_" },
```

This version seems to fail to load on the latest LTS version of Jenkins, which causes issues.

```
2025-11-18 12:21:47.045+0000 [id=51]	SEVERE	jenkins.InitReactorRunner$1#onTaskFailed: Failed Loading plugin Pipeline: API v1398.v67030756d3fb_ (workflow-api)
java.io.IOException: Failed to load: Pipeline: API (workflow-api 1398.v67030756d3fb_)
[CRLF]>  - Jenkins (2.532) or higher required
	at hudson.PluginWrapper.resolvePluginDependencies(PluginWrapper.java:1020)
	at hudson.PluginManager$2$1$1.run(PluginManager.java:590)
	at org.jvnet.hudson.reactor.TaskGraphBuilder$TaskImpl.run(TaskGraphBuilder.java:175)
	at org.jvnet.hudson.reactor.Reactor.runTask(Reactor.java:304)
	at jenkins.model.Jenkins$5.runTask(Jenkins.java:1151)
	at org.jvnet.hudson.reactor.Reactor$2.run(Reactor.java:221)
	at org.jvnet.hudson.reactor.Reactor$Node.run(Reactor.java:120)
	at jenkins.security.ImpersonatingExecutorService$1.run(ImpersonatingExecutorService.java:68)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
```

Version of Jenkins installed: `2.528.2`

---

```suggestion
        { "name": "workflow-api", "version": "1384.vdc05a_48f535f" },
```

Using this instead seems to work.